src/app/modules/gb-data-selection-module/accordion-components/gb-export/gb-export.component.ts

```diff
@@ -136,6 +138,28 @@ export class GbExportComponent implements OnInit {
       );
   }
 
+  cancelExportJob(job) {
+    job.isInDisabledState = true;
+    this.resourceService.cancelExportJob(job.id)
+      .subscribe(
+        response => {
+          this.updateExportJobs();
+        },
+        err => console.error(err)
+      );
+  }
+
+  archiveExportJob(job) {
+    job.isInDisabledState = true;
+    this.resourceService.archiveExportJob(job.id)
+      .subscribe(
+        response => {
+          this.updateExportJobs();
+        },
+        err => console.error(err)
```

Comment:
I see that poor error handling is currently standard in Glowing Bear. We should fix that at some point.